docs/source/guides/speed.rst

```diff
@@ -67,19 +52,21 @@ Refer to :doc:`Advanced GPU Optimized Training for more details <../advanced/adv
 
 Prefer DDP over DP
 ^^^^^^^^^^^^^^^^^^
-:class:`~pytorch_lightning.strategies.DataParallelStrategy` performs three GPU transfers for EVERY batch:
+
+:class:`~pytorch_lightning.strategies.dp.DataParallelStrategy` performs 3 GPU transfers for EVERY batch:
```

Comment:
I think we should include a gif of both of them. @edenlightning had an animation for this.

Reply:
pinged her on slack.

Reply:
added!